packages/core/contracts/optimistic-asserter/implementation/escalation-manager/DisputeLimitingEscalationManager.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./BaseEscalationManager.sol";
+import "../../interfaces/OptimisticAsserterInterface.sol";
+
+// This Escalation Manager blocks all assertions till the blocking dispute is resolved by Oracle. In order to avoid
+// interference among different applications this Escalation Manager allows assertions only from one requesting contract.
```

Comment:
```suggestion
// This Escalation Manager blocks all assertions till the blocking dispute is resolved by Oracle. In order to avoid
// interference among different applications this Escalation Manager allows assertions only from one requesting contract.
This is useful to create a system where only one assertion dispute can occur at a time. 
```